ChangeLog

```diff
@@ -23,6 +23,9 @@ Changes
      Contributed by Mathieu Briand.
    * Fix typo in a comment ctr_drbg.c. Contributed by Paul Sokolovsky.
    * Remove support for the library reference configuration for picocoin.
+   * Verify that when (f_send, f_recv and f_recv_timeout) send or receive 
```

Comment:
Suggestion: Change to `Double-check that the underlying transport's send and receive functions f_send, f_recv and f_recv_timeout do not report to have sent or received more than what was requested by the library. Raised by Sam O'Connor in #1245.`